src/definitions/behaviors/api.js

```diff
@@ -117,7 +117,7 @@
                             $module
                                 .on(triggerEvent + eventNamespace, module.event.trigger)
                             ;
-                        } else if (settings.on == 'now') {
+                        } else if (settings.on === 'now') {
```

Comment:
@lubber-de just checking, am I right here the change is functionally equivalent?

Reply:
String comparision should always be safe using === or am i not getting the question? :)

Reply:
see the actual/old code, there is `==` in many places

Reply:
please see the last commit - `==` must be reviewed very carefully case by case

these are true:
- `0 == '0'`
- `0 == ''`
- `null == undefined`

these are false:
- `null == ''`
- `null == 0`
- `undefined == ''`
- `undefined == 0`

thus we can autofix only `==` to `===` only for non-numerical strings

with this in mind, that is how I crafted this PR

Reply:
Yes, thats what i meant by string comparison. Your research is (again) perfectly summarized 🙂